tox.ini

```diff
@@ -0,0 +1,18 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
```

Comment:
Given that we're already running flake in the CI file and also use Github Actions jobs to run tests on multiple Python versions, do we really need tox here? Ideally I'd like to keep the number of dependencies as lean as possible so that we don't duplicate information and require new developers on the project to learn fewer new tools.

Reply:
The reason I added Tox was that it makes running tests locally a lot simpler since it deals with all the details of versions of software and so on; so I would prefer to keep it around unless you're very much against it. My preferred option is to just use Tox in the CI and drop everything else. Then all testing can be configured in tox.ini.